src/app/view/components/_common/searchable-dropdown/component/searchable-dropdown.component.ts

```diff
@@ -86,7 +89,7 @@ export class SearchableDropdownComponent implements ControlValueAccessor, OnChan
     propagateChange = (_: any) => {};
 
     ngOnChanges(change: SimpleChanges): void {
-        if (change.placeholder && change.placeholder.currentValue) {
+        if (this.usePlaceholder(change)) {
```

Comment:
If this method is checking if what is change is the placeholder why are we sending the whole change instead of change.placeholder?